lazy.config.ts

```diff
@@ -43,82 +43,86 @@ export function generateSharedTasks(bublic: '<rootDir>' | '<rootDir>/bublic') {
 				inputs: ['package.json', `${bublic}/scripts/refresh-assets.ts`, `${bublic}/assets/**/*`],
 			},
 		},
-		'build:types': {
+		'build-types': {
 			execution: 'top-level',
 			baseCommand: `tsx ${bublic}/scripts/typecheck.ts`,
 			cache: {
 				inputs: {
-					include: [
-						'{,bublic/}packages/*/src/**/*.{ts,tsx}',
-						'{,bublic/}{apps,scripts,e2e}/**/*.{ts,tsx}',
-						'{,bublic/}{apps,packages}/*/tsconfig.json',
-						'{,bublic/}{scripts,e2e}/tsconfig.json',
-					],
-					exclude: ['**/dist*/**/*.d.ts'],
+					include: ['<allWorkspaceDirs>/**/*.{ts,tsx}', '<allWorkspaceDirs>/tsconfig.json'],
+					exclude: ['<allWorkspaceDirs>/dist*/**/*', '<allWorkspaceDirs>/api/**/*'],
 				},
+				outputs: ['<allWorkspaceDirs>/*.tsbuildinfo', '<allWorkspaceDirs>/.tsbuild/**/*'],
```

Comment:
@SomeHats i added this new glob macro `<allWorkspaceDirs>` – really cleans a lot of stuff up.